debezium-embedded/src/test/java/io/debezium/embedded/AbstractConnectorTest.java

```diff
@@ -640,6 +781,13 @@ protected void assertNoRecordsToConsume() {
         assertThat(consumedLines.isEmpty()).isTrue();
     }
 
+    /**
+     * Assert that there are only transaction topic records to be consumed.
+     */
+    protected void assertOnlyTransactionRecordsToConsume() {
+        consumedLines.iterator().forEachRemaining(r -> assertThat(r.topic()).endsWith(".transaction"));
```

Comment:
SHould not the assert call `isTransactionRecord` method instead of chekcing the topic name?